diff --git a/node_modules/weasel-words/weasel.js b/node_modules/weasel-words/weasel.js
index a6bc8c4..7d8f5a1 100644
--- a/node_modules/weasel-words/weasel.js
+++ b/node_modules/weasel-words/weasel.js
@@ -37,6 +37,7 @@ var re = new RegExp('\\b(' + weasels.join('|') + ')\\b', 'gi');
 
 module.exports = function (text, opts) {
   var suggestions = [];
+  var match;
   while (match = re.exec(text)) {
     var weasel = match[0].toLowerCase();
     if (exceptions.indexOf(weasel) === -1 ||
